LoopKit/DeviceManager/DeviceManager.swift

```diff
@@ -53,4 +58,9 @@ public extension DeviceManager {
     var localizedTitle: String {
         return type(of: self).localizedTitle
     }
+    
+    var deviceManagerInstanceIdentifier: String {
```

Comment:
Add a comment that this is *not* currently unique per instance, but eventual intention is that it is unique per instance.